app/javascript/order_screen.js

```
@@ -247,6 +287,268 @@ document.addEventListener('turbo:load', () => {
+        onDropOnFolder(evt, folder) {
+          evt.preventDefault();
+          evt.stopPropagation();
+          if (!this.draggedItem || !folder) return;
+          
+          const productPrice = this.draggedItem;
+          const folderId = parseInt(folder.id);
+          
+          const productsInFolder = this.productPrices.filter(pp => pp.product_price_folder_id == folderId);
+          let maxPosition = -1;
+          productsInFolder.forEach(pp => {
+            if (typeof pp.position === 'number' && pp.position > maxPosition) {
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)